### DISCUSSION TITLE
Custom admin area info

### DISCUSSION THREAD

**Discussion by sbpowell** [swellstores/community#562] [Help]
I'm wondering if it's possible to create a notification on the Account page that shows the value of a custom field but otherwise isn't interactable. Currently we have a field that's updated, but it can be edited in the account page and it would be nicer if it was distinct and separate from Details. Thanks in advance.

**Comment by ericingram** [comment]
Are you using the app framework? If so then you can try adding `readonly: true` to the top-level field

**Comment by logeshswell** [comment]
@sbpowell I couldn't get your query. Can you please elaborate with the screenshots if possible?
1. What notification do you want to create?
2. What do you mean Account page?
